plugins/modules/rulebook_activation.py

```diff
@@ -539,6 +551,27 @@ def main() -> None:
         except EDAError as e:
             module.fail_json(msg=f"Failed to delete rulebook activation: {e}")
 
+    if restart:
+        try:
+            while activation["status"] not in ["running"]:
```

Comment:
This validation is business logic that replaces the backend logic and the module should not do it. If the activation can not be restarted is the backend who determines it. In fact already accepts restarts in other statuses.